zebra-state/src/service/finalized_state/zebra_db/shielded.rs

```diff
@@ -355,6 +428,102 @@ impl DiskWriteBatch {
             self.zs_insert(&orchard_tree_cf, height, trees.orchard);
         }
 
+        if let Some(subtree) = trees.sapling_subtree {
+            let (subtree_end, subtree_start) = subtree.bounds();
+
+            let start_height = if subtree_end.index == 0 {
+                // the first entry after Height(0) should be the height of the first Sapling note
+                let (start_height, _) = zebra_db
+                    .sapling_tree_by_height_range(Height(1)..)
```

Comment:
Let's use the height of the genesis block to make the code consistent with the comment above:
```suggestion
                    .sapling_tree_by_height_range(Height(0)..)
```

Reply:
It's skipping start boundaries for now, but I think the code was right and the comment was wrong in this case.